configs/foreman/1.20.yaml

```diff
@@ -0,0 +1,383 @@
+---
+:project: foreman
+:github_org: theforeman
+:releases:
+  :1.20.0:
+    :redmine_version_id: 885
+:mash_scripts:
+  - foreman-1.20-rhel7-dist
+  - foreman-plugins-1.20-rhel7-dist
+  - foreman-client-1.20-rhel7
+  - foreman-client-1.20-rhel6
+  - foreman-client-1.20-rhel5
+  - foreman-client-1.20-fedora27
+  - foreman-client-1.20-fedora28
+  - foreman-client-1.20-sles11
+  - foreman-client-1.20-sles12
+:repos: []
+:gpg_key: 565EA533
+:tags:
+  - name: foreman-1.20-nonscl-rhel7
+    based_off: foreman-nightly-nonscl-rhel7
+    helper_tags:
+      foreman-1.20-nonscl-rhel7-override: null
+    build_target: foreman-1.20-nonscl-rhel7-build
+    build_package_group_source_tag: foreman-nightly-nonscl-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-1.20-nonscl-rhel7-build:
+        foreman-1.20-nonscl-rhel7-override: 0
+      foreman-1.20-nonscl-rhel7-override:
+        foreman-1.20-nonscl-rhel7: 0
+    external_repos:
+      - epel-7
+      - puppetlabs-pc1-rhel-7
+      - centos-7-server-updates
+      - centos-7-server
+  - name: foreman-1.20-rhel7
+    based_off: foreman-nightly-rhel7
+    helper_tags:
+      foreman-1.20-rhel7-override: null
+    build_target: foreman-1.20-rhel7-build
+    build_package_group_source_tag: foreman-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-1.20-rhel7-build:
+        foreman-1.20-rhel7-override: 0
+        tfm-ror52-rhel7: 2
+        foreman-1.20-nonscl-rhel7: 10
+      foreman-1.20-rhel7-override:
+        foreman-1.20-rhel7: 0
+    external_repos:
+      - epel-7
+      - centos-sclo-rh-rhel-7
+      - centos-sclo-sclo-rhel-7
+      - centos-7-server-updates
+      - centos-7-server
+  - name: foreman-plugins-1.20-nonscl-rhel7
+    based_off: foreman-plugins-nightly-nonscl-rhel7
+    helper_tags:
+      foreman-plugins-1.20-nonscl-rhel7-override: null
+    build_target: foreman-plugins-1.20-nonscl-rhel7-build
+    build_package_group_source_tag: foreman-plugins-nightly-nonscl-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-plugins-1.20-nonscl-rhel7-build:
+        foreman-plugins-1.20-nonscl-rhel7-override: 0
+        foreman-1.20-rhel7: 15
+        foreman-1.20-nonscl-rhel7: 10
+      foreman-plugins-1.20-nonscl-rhel7-override:
+        foreman-plugins-1.20-nonscl-rhel7: 0
+    external_repos:
+      - epel-7
+      - centos-7-server-updates
+      - centos-7-server
+      - rhel-7.0-server-optional
```

Comment:
In theory it shouldn't, however, nightly was built with this still on this tag so I am inclined not to incur any potential issues in the release branch since it has not been vetted in nightly.

Reply:
Could we update nightly so we won't rediscover the same issue with 1.21? :)

Reply:
Oh, you did so already :)

Reply:
I did in this PR. I can split that out if you prefer.

Reply:
No, it's fine to do so in this PR. I was just too trigger happy with my comment.